fighters/palutena/src/acmd/aerials.rs

```diff
@@ -89,13 +92,50 @@ unsafe fn palutena_attack_air_f_game(fighter: &mut L2CAgentBase) {
     
 }
 
+#[acmd_script( agent = "palutena", script = "effect_attackairf", category = ACMD_EFFECT, low_priority )]
+unsafe fn palutena_attack_air_f_effect(agent: &mut L2CAgentBase) {
+    let lua_state = agent.lua_state_agent;
+    let boma = agent.boma();
+    frame(lua_state, 3.0);
+    if is_excute(agent) {
+        EFFECT_FOLLOW(agent, Hash40::new("palutena_wand_light_trace"), Hash40::new("stick"), 0, 8.65, 0, 0, 0, 0, 1, true);
+        LAST_EFFECT_SET_COLOR(agent, 2.0, 0.03, 0.01);
+        EFFECT_FOLLOW(agent, Hash40::new("palutena_wand_light2"), Hash40::new("stick"), 0, 8.65, 0, 0, 0, 0, 1, true);
+        LAST_EFFECT_SET_COLOR(agent, 2.0, 0.03, 0.01);
+    }
+    frame(lua_state, 9.0);
+    if sv_animcmd::get_value_float(lua_state, *SO_VAR_FLOAT_LR) < 0.0 {
```

Comment:
This block (and others like it throughout this PR) can be simplified as such:

```rs
if is_excute(fighter) {
    let facing = PostureModule::lr(boma);
    EFFECT_FOLLOW_NO_STOP(agent, Hash40::new("sys_flash"), Hash40::new("top"), 4.0 * facing, 7.5, 17, 0, 0, 0, 1, true);
    LAST_EFFECT_SET_RATE(agent, 1.3);
    EFFECT_FOLLOW(agent, Hash40::new("sys_attack_speedline"), Hash40::new("top"), 0, 8, 4, 0, 0, 0, 0.9, true);
}
```

Reply:
done